Makefile

```diff
@@ -46,12 +47,16 @@ local-overrides.yaml:
 #
 
 .PHONY: osp_full
-osp_full: local_requirements prepare_host network install_stack prepare_stack
+osp_full: local_requirements validations prepare_host network install_stack prepare_stack
 
 .PHONY: local_requirements
 local_requirements: inventory.yaml local-overrides.yaml
 	$(ANSIBLE_CMD) playbooks/local_requirements.yaml
 
+.PHONY: validations
+validations: inventory.yaml local-overrides.yaml
```

Comment:
once https://github.com/shiftstack/dev-install/pull/60 merges, we'll need to change `local-overrides.yaml` to `$(overrides)`.